packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -447,15 +612,31 @@ class TextSelectionOverlay {
     final Size handleSize = selectionControls!.getHandleSize(
       renderObject.preferredLineHeight,
     );
+
     _dragEndPosition = details.globalPosition + Offset(0.0, -handleSize.height);
+    final TextPosition position = renderObject.getPositionForPoint(_dragEndPosition);
+
+    _selectionOverlay.showMagnifier(MagnifierOverlayInfoBearer._fromRenderEditable(
```

Comment:
Nit: why two new lines?

Reply:
wdym two new lines? do you mean 618 should be deleted? I tried to separate each logical "step", like prepare the data than execute the update. If you think it looks better without the new line then I can del them

Reply:
Oh in the `files changed` tab it came out as two new lines instead of one. You can disregard any of my `two new lines` if its just one.